pkg/trace/config/config.go

```diff
@@ -338,6 +338,7 @@ type AgentConfig struct {
 	ExtraAggregators       []string      // DEPRECATED
 	PeerServiceAggregation bool          // enables/disables stats aggregation for peer.service, used by Concentrator and ClientStatsAggregator
 	ComputeStatsBySpanKind bool          // enables/disables the computing of stats based on a span's `span.kind` field
+	CustomTags             []string      // [EXPERIMENTAL] additional tags to use for stats aggregation
```

Comment:
Should we reuse the `ExtraAggregators` field? Perhaps its unsafe to do? I'm worried about calling these `Custom` tags because we are not yet ready to support things that are "custom".

Reply:
I wondered about this but then thought it could be unsafe for customers who may have old `ExtraAggregators` config lying around?